app/models/health/cha_saver.rb

```diff
@@ -40,13 +40,17 @@ def update
         # The CHA QA actually requires both the CHA and the SSM, so check both
         # also done in the SsmSaver so it can be done in either order
         if @complete && @create_qa && @cha.patient.recent_ssm_form&.completed_at.present?
-          qualifying_activity = setup_qualifying_activity
+          qualifying_activity = if @reviewed
+            setup_completed_qualifying_activity
```

Comment:
Is it possible to complete and review in one step?  Should we check to see if there's a development QA and create one if there isn't?